excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/ChallengeQuestionValidatorTest.java

```diff
@@ -0,0 +1,274 @@
+        String answer = "${OrganisationField.OrganisationName}|${OrganisationField.OrganisationID}:[CLAIMANT]," +
+            "${OrganisationField.OrganisationName}|${OrganisationField.OrganisationID}:[DEFENDANT]";
+        buildDefinitionDataItem(CASE_TYPE, FIELD_TYPE, "2", QUESTION_TEXT, DISPLAY_CONTEXT_PARAMETER_1, QUESTION_ID, answer,"questionId");
+        ChallengeQuestionTabEntity challengeQuestionTabEntity = challengeQuestionValidator.validate(parseContext, definitionDataItem);
+        assertValues(answer, challengeQuestionTabEntity, DISPLAY_CONTEXT_PARAMETER_1);
+    }
+
+    @Test
+    public void testAnswerFormatForSortExpression() {
+        String answer = "${OrganisationField.OrganisationName}|${OrganisationField.OrganisationID}:[CLAIMANT]";
+        buildDefinitionDataItem(CASE_TYPE, FIELD_TYPE, "2", QUESTION_TEXT, DISPLAY_CONTEXT_PARAMETER_2, QUESTION_ID, answer,"questionId");
+        ChallengeQuestionTabEntity challengeQuestionTabEntity = challengeQuestionValidator.validate(parseContext, definitionDataItem);
+        assertValues(answer, challengeQuestionTabEntity, DISPLAY_CONTEXT_PARAMETER_2);
+    }
+
+    @Test
+    public void testAnswerFormatForSmallestExpression() {
+        String answer = "${OrganisationField.OrganisationID}:[CLAIMANT]";
+        buildDefinitionDataItem(CASE_TYPE, FIELD_TYPE, "2", QUESTION_TEXT, DISPLAY_CONTEXT_PARAMETER_1, QUESTION_ID, answer,"questionId");
+        ChallengeQuestionTabEntity challengeQuestionTabEntity = challengeQuestionValidator.validate(parseContext, definitionDataItem);
+        assertValues(answer, challengeQuestionTabEntity, DISPLAY_CONTEXT_PARAMETER_1);
+    }
+
+    private void assertValues(String answer, ChallengeQuestionTabEntity challengeQuestionTabEntity, String displayContext) {
+
+        assertThat(challengeQuestionTabEntity.getAnswerField(), is(answer));
+        assertThat(challengeQuestionTabEntity.getCaseType().getReference(), is(CASE_TYPE));
+        assertThat(challengeQuestionTabEntity.getAnswerFieldType().getReference(), is(FIELD_TYPE));
+        assertThat(challengeQuestionTabEntity.getChallengeQuestionId(), is(QUESTION_ID));
+        assertThat(challengeQuestionTabEntity.getQuestionText(), is(QUESTION_TEXT));
+        assertThat(challengeQuestionTabEntity.getDisplayContextParameter(), is(displayContext));
+    }
+
+    @Test(expected = InvalidImportException.class)
+    public void failForCaseTypeValidation() {
+        try {
+            buildDefinitionDataItem("incorrectCaseType", FIELD_TYPE, "2", QUESTION_TEXT, DISPLAY_CONTEXT_PARAMETER_1, QUESTION_ID, ANSWERD,"questionId");
+            challengeQuestionValidator.validate(parseContext, definitionDataItem);
+        } catch (Exception exception) {
+            assertThat(exception.getMessage(), is("ChallengeQuestionTab Invalid Case Type value: incorrectCaseTypeit cannot be found the spreadsheet."));
+            throw exception;
+        }
+    }
+
+    @Test(expected = InvalidImportException.class)
```

Comment:
can we use assertThrows(...) instead of @Test(expected = InvalidImportException.class)